util/netcdf_generator.py

```diff
@@ -55,7 +55,8 @@ def _create_zip(self):
     def _filter_params(self, ds, params):
         missing_params = []
         params_to_filter = []
-        default_params = ['deployment', 'id', 'lat', 'lon', 'quality_flag']
+        # aggregation logic assumes the presence of a 'time' parameter, so do not allow it to get removed
```

Comment:
will 'time' also pickup time_15s and other virtual stream time parameters?

Reply:
Yes. 'time' will apply for the other virtual stream time parameters (currently 'met_timeflx', 'botsflu_time15s', and 'botsflu_time24h') because prior logic is reading the values from the time_parameter and storing it in the dataset.time attribute. See line 155 in stream_dataset.py.

Reply:
excellent!